seatunnel-engine/seatunnel-engine-common/src/main/java/org/apache/seatunnel/engine/common/loader/SeatunnelChildFirstClassLoader.java

```diff
@@ -27,17 +27,26 @@
 
 public class SeatunnelChildFirstClassLoader extends SeatunnelBaseClassLoader {
     private final String[] alwaysParentFirstPatterns;
-    private static final String SEATUNNEL_CLASS_PATTERN = "org.apache.seatunnel.";
+    private static final String[] DEFAULT_PARENT_FIRST_PATTERNS = new String[]{
```

Comment:
reference [link](https://github.com/apache/flink/blob/a91501f37262b461f7c645defe805686f7e274dd/flink-core/src/main/java/org/apache/flink/configuration/CoreOptions.java#L106)

Reply:
We can do this after merge into dev, cause ours option in https://github.com/apache/incubator-seatunnel/pull/2179/files#diff-be39eb1f82bdd74a53153699ceab3303d447421d555dcaf6dc60d53bbc60a180

Reply:
> We can do this after merge into dev, because our option in https://github.com/apache/incubator-seatunnel/pull/2179/files#diff-be39eb1f82bdd74a53153699ceab3303d447421d555dcaf6dc60d53bbc60a180

I agree with you.

Reply:
I agree